MODULE.bazel

```diff
@@ -5,7 +5,7 @@ module(
 )
 
 bazel_dep(name = "apple_support", version = "1.15.1")
-bazel_dep(name = "rules_swift", version = "1.18.0", repo_name = "build_bazel_rules_swift", max_compatibility_level = 2)
+bazel_dep(name = "rules_swift", version = "2.0.0", repo_name = "build_bazel_rules_swift")
```

Comment:
We should wait until 2.1.1 is approved, and use that, since it has a big bug fix in it.